bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/EnOceanBindingConstants.java

```diff
@@ -444,13 +445,6 @@
                     new EnOceanChannelDescription(new ChannelTypeUID(BINDING_ID, CHANNEL_SERVICECOMMAND),
                             CoreItemFactory.NUMBER)),
 
-            Map.entry(CHANNEL_STATUS_REQUEST_EVENT,
-                    new EnOceanChannelDescription(new ChannelTypeUID(BINDING_ID, CHANNEL_STATUS_REQUEST_EVENT), null,
-                            "", false, true)),
-            Map.entry(CHANNEL_SEND_COMMAND,
-                    new EnOceanChannelDescription(new ChannelTypeUID(BINDING_ID, CHANNEL_SEND_COMMAND),
-                            CoreItemFactory.SWITCH)),
```

Comment:
Is this removed without any replacement? Is this backward compatible?

Reply:
This change is backward compatible. A response is sent automatically after a request now. No need to use this channel any more. Should I create another PR in distro repo with a comment?

Reply:
As it's backward compatible, there's no need to generate a hint.